config/templates/claimPullRequest.md

```diff
@@ -0,0 +1,5 @@
+Hello @{commenter}!
+
+Thanks for your interest in Zulip! Please note that claiming pull requests is not supported. You can exclusively claim and submit pull requests for issues bearing the [help wanted](https://github.com/{repoOwner}/{repoName}/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee+label%3A%22help+wanted%22) label.
+
+If this is your first time here, we recommend reading our [guide for new contributors](https://zulip.readthedocs.io/en/latest/overview/contributing.html) before getting started.
```

Comment:
Let's make it:

---

Hello @{commenter}!

You have attempted to claim a pull request, which is someone else's proposed solution for an issue. Please review our [guide for new contributors](https://zulip.readthedocs.io/en/latest/overview/contributing.html) to learn how to find an issue to work on.